AUCTIONS_WINNERS.md

```diff
@@ -0,0 +1,135 @@
+<h1 style="text-align: center">WinnersData</h1>
+
+## Summary
+
+This is a guide to leverage substrate-api-sidecar's endpoints in order to extract the necessary data to know the winner's of an auction based on its `auctionIndex`.
+
+
+## Indexing
+
+To find the winner of a completed auction we will need to know the block number the auction ended at. Since Sidecar is a stateless API and auction info is wiped once it is over we need the block number to make historic queries to a state when the relevant data was stored (keep reading for details).
+
+This can be done by leveraging the `/experimental/paras/auctions/current` endpoint. 
+
+When there is an ongoing auction the return object will look like following below:
+
+```
+{
+  "at": {
+    "hash": "string",
+    "height": "string"
+  },
+  "beginEnd": "string",
+  "finishEnd": "string",
+  "phase": "opening",
+  "auctionIndex": "string",
+  "leasePeriods": [
+    "string"
+  ],
+  "winning": [
+    {
+      "bid": {
+        "accountId": "string",
+        "paraId": "string",
+        "amount": "string"
+      },
+      "leaseSet": [
+        "string"
+      ]
+    }
+  ]
+}
+```
+
+Important keys here are `finishEnd` which is the last block at which the auction will take place, `auctionIndex` which will act as your point of reference for that specific auction, and `leasePeriods` which are the available lease period indexes that may be bid on for the specific `auctionIndex`. 
+
+** Details of each key **
+
+`finishEnd`: This is the last block at which the auction will take place. Indexing it will allow you to query the block at which the auction ended. From that block you can extract the lease winning related events. (To query the block: GET `/blocks/{finishEnd}`.)
+
+`auctionIndex`: The unique identifier for the auction. 
+
+`leasePeriods`: These are the available lease period indexes that may be bid on for the specific `auctionIndex`. You can use this to create two seperate relationships explained below. 
+
+
+## Relationships
+
+With the information you are now tracking there are two main relationships to note. 
+
+1. `auctionIndex` => `leasePeriods` (Lease Periods that are actively being auctioned off)
+
+2. `paraId` => `leasePeriods` (Lease Periods that have been won by `paraId`).
```

Comment:
lower case is the way here you are correct, if we were mentioning its type though `LeasePeriod` then it would be capitalized. Probably where my confusion was.